src/include/wiredtiger.h.in

```diff
@@ -5522,6 +5522,11 @@ struct __wt_storage_source {
  */
 #define WT_PAGE_LOG_LSN_MAX UINT64_MAX
 
+/*!
+ * LSN used by Server during node step up process to avoid evicting pages.
+ */
+#define WT_DISAGG_START_LSN 1LL<<32
```

Comment:
Please use parenthesis around macros to avoid issues with macro substitution:
```c
#define WT_DISAGG_START_LSN (1LL<<32)
```

In this case, if you happen do write `WT_DISAGG_START_LSN + 5`, the compiler will see `1LL<<32 + 5` which evaluates to `1LL<<(32 + 5)` which is not what we wanted.

Parenthesis solve this issue for us.